src/store/reducers/interface.tsx

```diff
@@ -0,0 +1,24 @@
+export interface ListState {
+  tasks: Task[];
+}
+
```

Comment:
take it somewhere common interfaces you are using it multiple times

Reply:
this one is just for the reducer. it is not being used multiple times.